commons-numbers-complex-arrays/src/test/java/org/apache/commons/numbers/complex/arrays/ComplexListTest.java

```diff
@@ -152,9 +153,9 @@ void testAddIndexOutOfBoundExceptions() {
 
     @Test
     void testRemoveIndexOutOfBoundExceptions() {
+        List<Complex> objectList = generateList(2);
```

Comment:
Since a lot of the tests now use generateList() then addAll to a new ComplexList instance this should be refactored into a method: `private static ComplexList generateComplexList(int)`